blog-homepage/src/App.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import './App.css';
+import CardApp from './Card/CardApp'
+import missedArticles from './_data/missed-articles.json';
+import ForYouApp from './ForYou/ForYouApp';
+import yourArticles from './_data/your-articles.json';
+import Prototypes from 'prop-types';
+
+function App() {
+  const [{ title: fyTitle, description: fyDescription, image: fyImage, author: fyAuthor, postedDate: fyPostDate, minutesToRead: fyMinutesToRead, hasAudioAvailable }]= yourArticles;
+  const { name: fyAuthorName, image: fyAuthorImage} = fyAuthor;
+  const [{ title, description, image, author, postedDate, minutesToRead, hasAudioAvailable: hasMissedAudioAvailable }]= missedArticles;
+  const { name: authorName, image: authorImage} = author;
+  const months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+  
+  const forYouCard = yourArticles.map((card, idx) => 
+  <ForYouApp key = { idx }
+  imageUrl = { card.image } 
+    title = { card.title }
+    description = { card.description } 
+    authorImage = { card.author.image } 
+    authorName = { card.author.name } 
+    month = { months[new Date( card.postedDate ).getMonth()] }
+    todayDate = { new Date( card.postedDate).getUTCDate() }
+    minutesToRead = { card.minutesToRead } 
+    hasAudioAvailable = { card.hasAudioAvailable }
+    /> 
+);
+ 
+  const missedArticlesCard = missedArticles.map( (item, idx) => <CardApp key = { idx }
```

Comment:
Good use of components. One thing I would try would be formatting the code a little more so it's easier to read. Examples from class can give you ideas on how to do this.